product_variant_change_attribute_value/wizards/product_variant_attribute_value_wizard.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+
+class VariantAttributeValueWizard(models.TransientModel):
+    _name = "variant.attribute.value.wizard"
+    _description = "Wizard to change attriubtes on product variants"
+
+    product_ids = fields.Many2many(
+        "product.product", default=lambda self: self._default_product_ids()
+    )
+    product_variant_count = fields.Integer(compute="_compute_count")
+    product_template_count = fields.Integer(compute="_compute_count")
+    attributes_action_ids = fields.Many2many(
+        "variant.attribute.value.action",
+        relation="variant_attribute_wizard_attribute_action_rel",
+        default=lambda self: self._default_attributes_action_ids(),
+    )
+
+    def _default_product_ids(self):
+        return self.env["product.product"].browse(self._context.get("default_res_ids"))
```

Comment:
`default_res_ids` seems a custom key. Instead, simply set `default_product_ids` in the ctx and will work automatically w/out this method. All in all, it's just a default :wink: